powerlibs/aws/process_status_notifier/__init__.py

```diff
@@ -1,5 +1,44 @@
+from cached_property import cached_property
+
 from powerlibs.aws.sqs.publisher import SQSPublisher
 
 
 class ProcessStatusNotifier:
-    pass
+
+    def __init__(self, queue_name, process_name, topics_format='{process_name}__{status}', payload=None):
+        self.process_name = process_name
+        self.payload = payload if payload else dict()
+        self.queue_name = queue_name
+        self._load_basic_topics(topics_format)
+
+    @cached_property
+    def notifier(self):
+        return SQSPublisher()
+
+    def _load_basic_topics(self, topics_format):
+        self.status_topics = {
```

Comment:
Acho isso aqui um pouco mais "idiomático" (e não no bom sentido), mas evita umas paradas que eu acho que tendem a tornar o código confuso, como usar `append` em listas que, no término da execução do método, terão apenas um item cada.

```python
stati = 'started', 'finished', 'failed'
self.status_topics = {
    status: [topics_format.format(
        process_name=self.process_name,
        status=status
    )] for status in stati
}
```

Mas deixo a teu critério. Se for deixar do jeito que está, só peço duas coisas:

1- Trocar `list()` por `[]`.
2- Quebrar melhor as linhas do `topics_list.append`.